core/includes/module.inc

```diff
@@ -321,31 +321,19 @@ function module_load_install($module) {
  *
  * @return
  *   The name of the included file, if successful; FALSE otherwise.
+ *
+ * @deprecated since 1.22.0
```

Comment:
Nice 👍🏼  ...we also need a matching `@since` in the docblock of `backdrop_load_file()`.

Reply:
Thanks, didn't know about that: https://github.com/backdrop-ops/docs.backdropcms.org/issues/200